rootfs/etc/profile.d/prompt.sh

```diff
@@ -11,7 +14,14 @@ function prompter() {
 
 PROMPT_HOOKS+=("reload")
 function reload() {
-  kill -WINCH $$
+  local current_screen_size="${LINES}x${COLUMNS}"
+  # Detect changes in screensize
+  if [ "${current_screen_size}" != "${SCREEN_SIZE}" ]; then
+    echo "# Screen resized to ${current_screen_size}"
+    export SCREEN_SIZE=${current_screen_size}
+    # Instruct shell that window size has changed to ensure lines wrap correctly
+    kill -WINCH $$
+  fi
```

Comment:
The more I look at this, the more I don't think any of it should be necessary.
bash should be handling all this when jobs exit, and `checkwinsize` is `on`, which it is.